context/AuthContext.tsx

```
@@ -0,0 +1,282 @@
+
+        await AsyncStorage.setItem("user", JSON.stringify(userData));
+        if (childData) {
+          await AsyncStorage.setItem("child", JSON.stringify(childData));
+          setChild(childData);
+        }
+
+        setUser(userData);
+        router.replace("/(tabs)");
+      }
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)